container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend.go

```diff
@@ -473,6 +473,18 @@ func (backend *DockerKurtosisBackend) DestroyLogsCollectorForEnclave(ctx context
 	return nil
 }
 
+func (backend *DockerKurtosisBackend) DestroyLogsStorage(ctx context.Context) error {
```

Comment:
I don't see any call to this method.  I could be wrong. is it used?

Reply:
I was initially going to use this `DestroyLogsStorage` to remove logs during a `kurtosis clean -a` from the CLI but realized that wasn't possible because the volume can't be removed because the running engine container is still using it even after a `clean` - so I opted to use `RemoveEnclaveLogs` from the engine to remove logs instead. I'll remove it on the principle of dead code is dangerous.